pages/desktop/consumer_pages/reviews.py

```diff
@@ -27,24 +27,29 @@ def __init__(self, testsetup, app_name=False):
     @property
     def reviews(self):
         """Returns review object with index."""
-        return [self.ReviewSnippet(self.testsetup, web_element)
+        return [self.Review(self.testsetup, web_element)
                 for web_element in self.selenium.find_elements(*self._review_locator)]
 
-    @property
-    def logged_in_users_review(self):
+    def get_review_for_user(self, user_name):
         for review in self.reviews:
-            if review.find_element(*review._delete_review_locator):
+            if review.author == user_name:
                 break
```

Comment:
Honestly might be due to #toddlerlife #nosleep effecting me, should `return review` be moved into this `if` block?